### PR TITLE
Fix data table custom sort functionality

### LINKED ISSUE #210: Sort is broken for some columns
The sort function for some columns does not seem to be working properly:

-Under Cases tab: HTAN Participant ID, Atlas Name, Primary Diagnosis, Age at Diagnosis
-Under Biospecimens tab: HTAN Biospecimen ID, Atlas Name, Biospecimen Type

### PR DESCRIPTION
**Pull request by onursumer**
Fix #210

### CODE REVIEW

File: components/BiospecimenTable.tsx [hunk 1]
```diff
@@ -9,21 +12,6 @@ interface IBiospecimenTableProps {
     synapseAtlases: Atlas[];
 }
 
-const sortFunction = (rows: any[], field: string, direction: any) => {
```
Comment (onursumer) [inline components/BiospecimenTable.tsx:12]: This was only working for `HTANBiospecimenID` and only in ascending order. When adding a custom `sortFunction` we need to make sure that we sort the rows for all fields and for all directions.

File: components/CaseTable.tsx [hunk 2]
```diff
@@ -9,21 +12,6 @@ interface ICaseTableProps {
     synapseAtlases: Atlas[];
 }
 
-// we need sort by participant id which takes the form HTA[integer]_[integer]
-const sortFunction = (rows: any[], field: string, direction: any) => {
```
Comment (onursumer) [inline components/CaseTable.tsx:13]: Same as BiospecimenTable, this was only working for a single field only in ascending direction.

File: components/CaseTable.tsx [hunk 3]
```diff
@@ -33,10 +21,11 @@ export const CaseTable: React.FunctionComponent<ICaseTableProps> = (props) => {
             selector: 'HTANParticipantID',
             wrap: true,
             sortable: true,
+            sortFunction: sortByHtanParticipantId,
         },
         {
             name: 'Atlas Name',
-            cell: (sample: Entity) => {
+            selector: (sample: Entity) => {
```
Comment (onursumer) [inline components/CaseTable.tsx:28]: default sort functionality requires a selector.

File: components/BiospecimenTable.tsx [hunk 4]
```diff
@@ -35,11 +23,11 @@ export const BiospecimenTable: React.FunctionComponent<IBiospecimenTableProps> =
             selector: 'HTANBiospecimenID',
             wrap: true,
             sortable: true,
+            sortFunction: sortByBiospecimenId,
         },
         {
             name: 'Atlas Name',
-
-            cell: (sample: Entity) => {
+            selector: (sample: Entity) => {
```
Comment (onursumer) [inline components/BiospecimenTable.tsx:30]: default sort functionality requires a selector.

**Review by adamabeshouse — APPROVED** [review]

**Pull request merged**